<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1071</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Recherche des clients</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>351</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>RECHERCHER UN CLIENT</string>
      </property>
      <widget class="QPushButton" name="pushButton_recherche">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>31</y>
         <width>333</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>40</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Prénom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_prenom">
            <property name="text">
             <string extracomment="Mon prenom"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>60</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_nom"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Numéro ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_numeroID"/>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_id">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>120</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>20</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Date début</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEdit_debutRecherche"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>27</number>
              </property>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Date fin</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEdit_finRecherche">
                <property name="dateTime">
                 <datetime>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                  <year>2000</year>
                  <month>1</month>
                  <day>1</day>
                 </datetime>
                </property>
                <property name="maximumDate">
                 <date>
                  <year>7999</year>
                  <month>12</month>
                  <day>21</day>
                 </date>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_dates">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>661</width>
        <height>561</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>641</width>
         <height>491</height>
        </rect>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>510</y>
         <width>201</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="pushButton_modifier">
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_supprimer">
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Affichage des ressources</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>871</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>RESSOURCES</string>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>831</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>390</y>
         <width>199</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="pushButton_modifier_ressources">
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_supprimer_ressources">
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>520</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Plannification</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuAjouter">
     <property name="title">
      <string>Ajouter</string>
     </property>
     <addaction name="actionClient"/>
     <addaction name="actionRessource"/>
     <addaction name="actionDivers"/>
    </widget>
    <addaction name="menuAjouter"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClient"/>
   <addaction name="actionRessource"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/icns/if_close_32391.icns</normaloff>:/rec/icns/if_close_32391.icns</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionClient">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/icns/if_contact-new_23203.icns</normaloff>:/rec/icns/if_contact-new_23203.icns</iconset>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRessource">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/icns/if_group_full_add_103573.icns</normaloff>:/rec/icns/if_group_full_add_103573.icns</iconset>
   </property>
   <property name="text">
    <string>Ressource</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/icns/if_Help_1493288.icns</normaloff>:/rec/icns/if_Help_1493288.icns</iconset>
   </property>
   <property name="text">
    <string>A propos</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDivers">
   <property name="text">
    <string>Divers</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_prenom</tabstop>
  <tabstop>pushButton_modifier_ressources</tabstop>
  <tabstop>lineEdit_numeroID</tabstop>
  <tabstop>dateEdit_debutRecherche</tabstop>
  <tabstop>dateEdit_finRecherche</tabstop>
  <tabstop>pushButton_recherche</tabstop>
  <tabstop>pushButton_modifier</tabstop>
  <tabstop>pushButton_supprimer</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton_supprimer_ressources</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_nom</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
